dockerfy/dockers/back/Dockerfile

```diff
@@ -1,9 +1,14 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 LABEL maintainer="ravada@telecos.upc.edu"
 LABEL description="Ravada Backend + KVM"
 
-RUN apt-get update \
- && apt-get install -y -q --no-install-recommends \
+RUN apt-get update
+ENV TZ=Europe/Madrid
```

Comment:
It's not necessary to define it in the Dockerfile. TZ is already set in the [.env](https://github.com/UPC/ravada/blob/develop/dockerfy/.env) file to allow you to define your own time zone.